envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/snapshot/resource/listeners/filters/AccessLogFilter.kt

```diff
@@ -55,6 +55,7 @@ class AccessLogFilter(
                             .putFields("request_path", stringValue("%REQ(:PATH)%"))
                             .putFields("response_code", stringValue("%RESPONSE_CODE%"))
                             .putFields("response_flags", stringValue("%RESPONSE_FLAGS%"))
+                            .putFields("request_id", stringValue("%REQ(X-REQUEST-ID)%"))
```

Comment:
is it possible to add a test for this?

Reply:
Yes it is but it would be awfull. here we set only fields for protobuff